packages/fxa-settings/src/components/ConfirmWithLink/index.tsx

```diff
@@ -40,8 +40,8 @@ const ConfirmWithLink = ({
         headingTextFtlId={confirmWithLinkPageStrings.headingFtlId}
       />
 
-      {resendStatus === ResendStatus['sent'] && <ResendEmailSuccessBanner />}
-      {resendStatus === ResendStatus['error'] && errorMessage && (
+      {resendStatus === ResendStatus.sent && <ResendEmailSuccessBanner />}
```

Comment:
Nice, I like this better. I noticed in some other ticket but we are a little inconsistent with how we're displaying the banners/setting state for them in at least one or two places, but not sure if it's worth another follow up (and unrelated to your changes here, just talking out loud).

Reply:
True. I'll add a comment in the ticket to update the banner styles, that might be a logical time to review how we implement the banner and manage it's state.